.circleci/config.yml

```diff
@@ -241,7 +241,14 @@ jobs:
       - run:
           name: "Run Fuzzer Tests"
           command: |
-            make fuzzertest NUM_THREADS=10
+            # Run fuzzer using the built executable - we do this instead of make
+            # since currently make fuzzer-test tends to rebuild the project.
```

Comment:
fuzzer-test -> fuzzertest ?

Reply:
Yes; wonder why unittest and fuzzertest dont have hyphens but header-fix, format-fix, header-check etc do .. 😄